### PR TITLE
OpenSSL update to Noam's FIPS blog post

### PR DESCRIPTION
**Pull request by sej7278**
Just adds a footnote about the OpenSSL certificate getting delivered just after we published the blog

### CODE REVIEW

**Review by bennyvasquez — APPROVED** [review]
Once @Noam-Alum okays this, I'll hit the merge button.

**Comment by Noam-Alum** [comment]
> Once @Noam-Alum okays this, I'll hit the merge button. 

No problem at all, go ahead.

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**
